From 21c535326436df93fd61c9bd029b8716d1ab94d0 Mon Sep 17 00:00:00 2001
From: Vladimir Oltean <vladimir.oltean@nxp.com>
Date: Fri, 29 Nov 2019 03:10:23 +0200
Subject: [PATCH] arm64: dts: fsl: Drop "compatible" string from Felix switch

Since Felix is not a platform device but a PCI device, the "compatible"
string serves no purpose. The device driver is found by matching the PCI
device/vendor ID to the ENETC PF.

Signed-off-by: Vladimir Oltean <vladimir.oltean@nxp.com>
---
 arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi | 1 -
 1 file changed, 1 deletion(-)

--- a/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi
@@ -788,7 +788,6 @@
 				little-endian;
 			};
 			switch@0,5 {
-				compatible = "mscc,felix-switch";
 				reg = <0x000500 0 0 0 0>;
 				/* IEP INT_B */
 				interrupts = <GIC_SPI 95 IRQ_TYPE_LEVEL_HIGH>;
